nifi-nar-bundles/nifi-hive-bundle/nifi-hive-processors/src/main/java/org/apache/nifi/dbcp/hive/HiveConnectionPool.java

```diff
@@ -71,6 +72,9 @@
 @RequiresInstanceClassLoading
 @Tags({"hive", "dbcp", "jdbc", "database", "connection", "pooling", "store"})
 @CapabilityDescription("Provides Database Connection Pooling Service for Apache Hive. Connections can be asked from pool and returned after usage.")
+@DynamicProperty(name = "The name of a Hive configuration property.", value = "The value of the given Hive configuration property.",
```

Comment:
I definitely appreciate your effort to add documentation that was missing from already-implemented features. However as it turns out, dynamic properties don't actually work for the Hive connection pools, so I wrote up [NIFI-8016](https://issues.apache.org/jira/browse/NIFI-8016) to cover this (and actually to change the behavior). Because I expect a behavior change, the documentation should be added as part of that Jira to explain what it does once correctly implemented.

Do you mind removing the doc from this PR? I can add this as part of NIFI-8016 (or you can if you like 😄 )

Reply:
I added the doc to #4676